website/content/docs/k8s/upgrade/index.mdx

```diff
@@ -68,43 +72,43 @@ certain Helm chart version.
 
 1. Update your local Helm repository cache:
 
-```bash
-helm repo update
-```
-
-1. List all available versions:
+  ```shell-session
+  $ helm repo update
+  ```
 
-```shell-session hideClipboard
-$ helm search repo hashicorp/consul --versions
-NAME            	CHART VERSION	APP VERSION	DESCRIPTION
-hashicorp/consul	0.24.1       	1.8.2      	Official HashiCorp Consul Chart
-hashicorp/consul	0.24.0       	1.8.1      	Official HashiCorp Consul Chart
-...
-```
+1. List all available versions. Here we can see that the latest version of `0.40.0`.
```

Comment:
```suggestion
1. List all available versions. Here you can observe that the latest version of `0.40.0`.
```

Use "we" when recommendations are being provided, otherwise, always address the user.